arrayfire/library.py

```diff
@@ -422,6 +422,14 @@ class CANNY_THRESHOLD(_Enum):
     MANUAL = _Enum_Type(0)
     AUTO_OTSU = _Enum_Type(1)
 
+class TOPK(_Enum):
+    """
+    Top-K function types
```

Comment:
topk function `filter options` ? may be, types is confusing I think.

Reply:
This is similar to what the enum is in C/C++. Let us keep this consistent.

Reply:
Sorry I misread the location of this comment. I agree with @9prady9 here about fixing the docs.

Reply:
@9prady9 @pavanky Done.